### PR TITLE
refactor(nameupdates): name updates to examples in frmw files

### PR DESCRIPTION
**Pull request by cathysiller**
JIRA: https://jira.rax.io/browse/FRMW-1267

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM

### CODE REVIEW

File: src/elements/Forms/docs/examples/rxSelect.simple.html [hunk 1]
```diff
@@ -1,24 +0,0 @@
...
-            <option value="thin">Thin (light and crispy)</option>
-            <option value="medium">Medium (perfect balance of flavor)</option>
-            <option value="thick">Thick (borderline jerky)</option>
-            <option value="crumbled">Crumbled (great on salads)</option>
-          </select>
-        </rx-input>
-      </rx-field-content>
-    </rx-field>
-  </rx-form-section>
-</form>
```
Comment (CITguy) [inline src/elements/Forms/docs/examples/rxSelect.simple.html]: I'm not sure this was meant to be removed. Please put back content but rename to `select.simple.html`

File: src/elements/Forms/docs/rxDatePicker.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 angular.module('demoApp')
-.controller('rxDatePickerCtrl', function ($scope) {
+.controller('datePickerCtrl', function ($scope) {
```
Comment (CITguy) [inline src/elements/Forms/docs/rxDatePicker.js]: Would this be better named as `datePicker.docs.js` with the controller named `datePickerDocsCtrl`?

File: src/elements/Forms/docs/rxCharacterCount.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 angular.module('demoApp')
-.controller('rxCharacterCountCtrl', function ($scope) {
+.controller('characterCountCtrl', function ($scope) {
```
Comment (CITguy) [inline src/elements/Forms/docs/rxCharacterCount.js]: Would this be better named as `characterCount.docs.js` with the controller named `characterCountDocsCtrl`?

File: src/elements/Forms/docs/rxSelect.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
 angular.module('demoApp')
-.controller('rxSelectCtrl', function ($scope) {
+.controller('selectCtrl', function ($scope) {
```
Comment (CITguy) [inline src/elements/Forms/docs/rxSelect.js]: Would this be better named as `select.docs.js` with the controller named `selectDocsCtrl`?

File: src/elements/Forms/docs/rxTimePicker.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
 angular.module('demoApp')
-.controller('rxTimePickerCtrl', function ($scope) {
+.controller('timePickerCtrl', function ($scope) {
```
Comment (CITguy) [inline src/elements/Forms/docs/rxTimePicker.js]: Would this be better named as `timePicker.docs.js` with the controller named `timePickerDocsCtrl`?

File: src/elements/Forms/scripts/rxSearchBox.js [hunk 6]
```diff
@@ -63,9 +63,9 @@ angular.module('encore.ui.elements')
             };
         },
         link: function (scope, element, attrs, controllers) {
-            var rxFloatingHeaderCtrl = controllers[1];
-            if (_.isObject(rxFloatingHeaderCtrl)) {
-                rxFloatingHeaderCtrl.update();
+            var floatingHeaderCtrl = controllers[1];
+            if (_.isObject(floatingHeaderCtrl)) {
+                floatingHeaderCtrl.update();
```
Comment (CITguy) [inline src/elements/Forms/scripts/rxSearchBox.js]: Please revert these changes. This is an internal reference to the `rxFloatingHeader` directive controller.

File: src/elements/Modals/docs/examples/modals.multi-view-modal.html [hunk 7]
```diff
@@ -1,7 +1,7 @@
 <p>
   <rx-modal-action
     classes="button"
-    controller="rxModalMultiViewCtrl"
+    controller="modalMultiViewCtrl"
```
Comment (CITguy) [inline src/elements/Modals/docs/examples/modals.multi-view-modal.html]: Please rename file to `modal.multiView.html`

File: src/elements/Modals/docs/examples/modals.multi-view-modal.js [hunk 8]
```diff
@@ -1,5 +1,5 @@
 angular.module('demoApp')
-.controller('rxModalMultiViewCtrl', function ($scope, $modalInstance, $timeout, rxNotify) {
+.controller('modalMultiViewCtrl', function ($scope, $modalInstance, $timeout, rxNotify) {
```
Comment (CITguy) [inline src/elements/Modals/docs/examples/modals.multi-view-modal.js]: Please rename file to `modal.multiView.js`

File: src/elements/Tables/docs/examples/table.paginate.api.html [hunk 9]
```diff
@@ -1,4 +1,4 @@
-<div ng-controller="rxPaginateApiCtrl">
+<div ng-controller="paginateApiCtrl">
```
Comment (CITguy) [inline src/elements/Tables/docs/examples/table.paginate.api.html]: Please rename file to `table.apiPagination.html` and controller to `tableApiPaginationCtrl`

File: src/elements/Tables/docs/examples/table.paginate.api.js [hunk 10]
```diff
@@ -1,5 +1,5 @@
 angular.module('demoApp')
-.controller('rxPaginateApiCtrl', function ($scope, $q, $timeout, $filter, rxPageTracker,
+.controller('paginateApiCtrl', function ($scope, $q, $timeout, $filter, rxPageTracker,
```
Comment (CITguy) [inline src/elements/Tables/docs/examples/table.paginate.api.js]: Please rename file to `table.apiPagination.js` and controller to `tableApiPaginationCtrl`.

File: src/elements/Tables/docs/examples/table.paginate.ui.html [hunk 11]
```diff
@@ -1,4 +1,4 @@
-<div ng-controller="rxPaginateUiCtrl">
+<div ng-controller="paginateUiCtrl">
```
Comment (CITguy) [inline src/elements/Tables/docs/examples/table.paginate.ui.html]: Please rename file to `table.uiPagination.html` and controller to `tableUiPaginationCtrl`

File: src/elements/Tables/docs/examples/table.paginate.ui.js [hunk 12]
```diff
@@ -1,5 +1,5 @@
 angular.module('demoApp')
-.controller('rxPaginateUiCtrl', function ($scope, rxPageTracker) {
+.controller('paginateUiCtrl', function ($scope, rxPageTracker) {
```
Comment (CITguy) [inline src/elements/Tables/docs/examples/table.paginate.ui.js]: Please rename file to `table.uiPagination.js` and controller to `tableUiPaginationCtrl`

File: src/elements/Tables/docs/examples/table.statusColumn.html [hunk 13]
```diff
@@ -1,5 +1,5 @@
 <table 
-  ng-controller="rxStatusColumnCtrl"
+  ng-controller="statusColumnCtrl"
```
Comment (CITguy) [inline src/elements/Tables/docs/examples/table.statusColumn.html]: Please rename controller to `tableStatusColumnCtrl`

File: src/elements/Tables/docs/examples/table.statusColumn.js [hunk 14]
```diff
@@ -1,5 +1,5 @@
 angular.module('demoApp')
-.controller('rxStatusColumnCtrl', function ($scope, rxStatusMappings, rxSortUtil) {
+.controller('statusColumnCtrl', function ($scope, rxStatusMappings, rxSortUtil) {
```
Comment (CITguy) [inline src/elements/Tables/docs/examples/table.statusColumn.js]: Same as above.

File: src/elements/Tables/docs/tables.docs.html [hunk 15]
```diff
@@ -191,7 +191,7 @@
 <rx-example name="table.paginate.ui"></rx-example>
 
 <rx-debug>
-  <div ng-controller="rxPaginateUiCtrl">
+  <div ng-controller="paginateUiCtrl">
```
Comment (CITguy) [inline src/elements/Tables/docs/tables.docs.html]: See above for name change.

File: src/elements/Tables/scripts/rxBatchActions.js [hunk 16]
```diff
@@ -70,7 +70,7 @@ angular.module('encore.ui.elements')
                 if (!scope.displayed) {
                     rxDOMHelper.find(element, '.batch-action-menu-container').addClass('ng-hide');
                 }
-                rxFloatingHeaderCtrl.update();
+                floatingHeaderCtrl.update();
```
Comment (CITguy) [inline src/elements/Tables/scripts/rxBatchActions.js]: Please revert changes to this file. These are internal references to the `rxFloatingHeader` directive controller.

File: src/elements/Tables/scripts/rxBulkSelectMessage.js [hunk 17]
```diff
@@ -64,20 +64,20 @@ angular.module('encore.ui.elements')
...
+                    floatingHeaderCtrl.update();
                 }
             };
             rxBulkSelectCtrl.registerForNumSelected(numSelectedChange);
 
             rxBulkSelectCtrl.registerForTotal(function (newTotal) {
                 scope.total = newTotal;
             });
-            rxFloatingHeaderCtrl.update();
+            floatingHeaderCtrl.update();
```
Comment (CITguy) [inline src/elements/Tables/scripts/rxBulkSelectMessage.js]: Please revert changes to this file. These are internal references to the `rxFloatingHeader` directive controller.

File: src/utilities/rxApply/docs/examples/rxApply.simple.html [hunk 18]
```diff
@@ -1,4 +1,4 @@
-<div ng-controller="ApplySimpleCtrl">
+<div ng-controller="rxApplySimpleCtrl">
```
Comment (CITguy) [inline src/utilities/rxApply/docs/examples/rxApply.simple.html]: Please rename file to `apply.simple.html` and controller to `applySimpleCtrl`.

File: src/utilities/rxApply/docs/examples/rxApply.simple.js [hunk 19]
```diff
@@ -1,5 +1,5 @@
 angular.module('demoApp')
-.controller('ApplySimpleCtrl', function ($scope, rxSelectFilter) {
+.controller('rxApplySimpleCtrl', function ($scope, rxSelectFilter) {
```
Comment (CITguy) [inline src/utilities/rxApply/docs/examples/rxApply.simple.js]: Please rename file to `apply.simple.js` and controller to `applySimpleCtrl`.

File: src/elements/Tables/docs/examples/floatingHeader.html [hunk 20]
```diff
@@ -1,4 +1,4 @@
-<div ng-controller="rxFloatingHeaderCtrl">
+<div ng-controller="floatingHeaderCtrl">
```
Comment (CITguy) [inline src/elements/Tables/docs/examples/floatingHeader.html]: Please rename file to `table.floatingHeader.html` and controller to `tableFloatingHeaderCtrl`.

File: src/elements/Tables/docs/examples/floatingHeader.js [hunk 21]
```diff
@@ -1,5 +1,5 @@
 angular.module('demoApp')
-.controller('rxFloatingHeaderCtrl', function ($scope) {
+.controller('floatingHeaderCtrl', function ($scope) {
```
Comment (CITguy) [inline src/elements/Tables/docs/examples/floatingHeader.js]: Please rename file to `table.floatingHeader.js` and controller to `tableFloatingHeaderCtrl`.

File: src/elements/Forms/docs/rxDatePicker.js (hunk 2 above)
Comment (cathysiller) [inline src/elements/Forms/docs/rxDatePicker.js]: Updated

File: src/elements/Forms/docs/rxCharacterCount.js (hunk 3 above)
Comment (cathysiller) [inline src/elements/Forms/docs/rxCharacterCount.js]: updated

File: src/elements/Forms/docs/rxSelect.js (hunk 4 above)
Comment (cathysiller) [inline src/elements/Forms/docs/rxSelect.js]: updated

File: src/elements/Forms/docs/rxTimePicker.js (hunk 5 above)
Comment (cathysiller) [inline src/elements/Forms/docs/rxTimePicker.js]: updated

File: src/elements/Forms/scripts/rxSearchBox.js (hunk 6 above)
Comment (cathysiller) [inline src/elements/Forms/scripts/rxSearchBox.js]: reverted

File: src/elements/Modals/docs/examples/modals.multi-view-modal.html (hunk 7 above)
Comment (cathysiller) [inline src/elements/Modals/docs/examples/modals.multi-view-modal.html]: updated

File: src/elements/Modals/docs/examples/modals.multi-view-modal.js (hunk 8 above)
Comment (cathysiller) [inline src/elements/Modals/docs/examples/modals.multi-view-modal.js]: updated

File: src/elements/Tables/docs/examples/table.paginate.api.html (hunk 9 above)
Comment (cathysiller) [inline src/elements/Tables/docs/examples/table.paginate.api.html]: updated

File: src/elements/Tables/docs/examples/table.paginate.api.js (hunk 10 above)
Comment (cathysiller) [inline src/elements/Tables/docs/examples/table.paginate.api.js]: updated

File: src/elements/Tables/docs/examples/table.paginate.ui.js (hunk 12 above)
Comment (cathysiller) [inline src/elements/Tables/docs/examples/table.paginate.ui.js]: updated

File: src/elements/Tables/docs/tables.docs.html (hunk 15 above)
Comment (cathysiller) [inline src/elements/Tables/docs/tables.docs.html]: updated

Comment (cathysiller) [inline src/elements/Tables/docs/tables.docs.html]: does line 191 need to be updated to 'table.uiPagination' @CITguy ?

File: src/elements/Tables/docs/examples/table.statusColumn.html (hunk 13 above)
Comment (cathysiller) [inline src/elements/Tables/docs/examples/table.statusColumn.html]: updated

File: src/elements/Tables/docs/examples/table.statusColumn.js (hunk 14 above)
Comment (cathysiller) [inline src/elements/Tables/docs/examples/table.statusColumn.js]: updated

File: src/elements/Tables/scripts/rxBatchActions.js (hunk 16 above)
Comment (cathysiller) [inline src/elements/Tables/scripts/rxBatchActions.js]: reverted

File: src/elements/Tables/scripts/rxBulkSelectMessage.js (hunk 17 above)
Comment (cathysiller) [inline src/elements/Tables/scripts/rxBulkSelectMessage.js]: reverted

File: src/utilities/rxApply/docs/examples/rxApply.simple.html (hunk 18 above)
Comment (cathysiller) [inline src/utilities/rxApply/docs/examples/rxApply.simple.html]: utilities -- fine with rx

File: src/utilities/rxApply/docs/examples/rxApply.simple.js (hunk 19 above)
Comment (cathysiller) [inline src/utilities/rxApply/docs/examples/rxApply.simple.js]: utilities -- fine with rx

File: src/elements/Tables/docs/examples/floatingHeader.html (hunk 20 above)
Comment (cathysiller) [inline src/elements/Tables/docs/examples/floatingHeader.html]: updated

File: src/elements/Tables/docs/examples/floatingHeader.js (hunk 21 above)
Comment (cathysiller) [inline src/elements/Tables/docs/examples/floatingHeader.js]: updated

File: src/elements/Forms/docs/rxCharacterCount.js [hunk 22]
```diff
@@ -1,5 +1,5 @@
 angular.module('demoApp')
-.controller('rxCharacterCountCtrl', function ($scope) {
+.controller('characterCountDocsCtrl', function ($scope) {
```
Comment (CITguy) [inline src/elements/Forms/docs/rxCharacterCount.js]: Please rename file to `characterCount.docs.js`

File: src/elements/Forms/docs/rxDatePicker.js [hunk 23]
```diff
@@ -1,5 +1,5 @@
 angular.module('demoApp')
-.controller('rxDatePickerCtrl', function ($scope) {
+.controller('datePickerDocsCtrl', function ($scope) {
```
Comment (CITguy) [inline src/elements/Forms/docs/rxDatePicker.js]: Please rename file to `datePicker.docs.js`

File: src/elements/Forms/docs/rxSelect.js [hunk 24]
```diff
@@ -1,5 +1,5 @@
 angular.module('demoApp')
-.controller('rxSelectCtrl', function ($scope) {
+.controller('selectDocsCtrl', function ($scope) {
```
Comment (CITguy) [inline src/elements/Forms/docs/rxSelect.js]: Please rename file to `select.docs.js`

File: src/elements/Forms/docs/rxTimePicker.js [hunk 25]
```diff
@@ -1,5 +1,5 @@
 angular.module('demoApp')
-.controller('rxTimePickerCtrl', function ($scope) {
+.controller('timePickerDocsCtrl', function ($scope) {
```
Comment (CITguy) [inline src/elements/Forms/docs/rxTimePicker.js]: Please rename file to `timePicker.docs.js`

File: src/elements/Modals/docs/Modals.docs.html [hunk 26]
```diff
@@ -20,10 +20,10 @@
   It works for cases when removing or deleting something needing a
   confirmation click, such as "Are You Sure you want to Remove this?"
 </p>
-<rx-example name="modals.multi-view-modal"></rx-example>
+<rx-example name="modals.multiView"></rx-example>
 
 <h3><rx-permalink>Disabled Modal</rx-permalink></h3>
-<rx-example name="modals.disabled-modal"></rx-example>
+<rx-example name="modals.disabledModal"></rx-example>
```
Comment (CITguy) [inline src/elements/Modals/docs/Modals.docs.html]: Please rename `modals.disabledModal.*` to `modals.disabled.*`. The extra `Modal` at the end is unnecessary.

File: src/elements/Modals/docs/Modals.docs.html [hunk 27]
```diff
@@ -71,7 +71,7 @@
     and build upon that.
   </li>
 </ul>
-<rx-example name="modals.delete-modal"></rx-example>
+<rx-example name="modals.deleteModal"></rx-example>
```
Comment (CITguy) [inline src/elements/Modals/docs/Modals.docs.html]: Please rename `modals.deleteModal.*` to `modals.delete.*`. The extra `Modal` at the end is unnecessary.

File: src/elements/Tables/docs/examples/table.uiPagination.html [hunk 28]
```diff
@@ -1,4 +1,4 @@
-<div ng-controller="rxPaginateUiCtrl">
+<div ng-controller="tableUiPaginationCtr">
```
Comment (CITguy) [inline src/elements/Tables/docs/examples/table.uiPagination.html]: Please correct misspelling (missing ending `l` in `Ctrl`).

File: src/elements/Tables/docs/examples/table.uiPagination.js [hunk 29]
```diff
@@ -1,5 +1,5 @@
 angular.module('demoApp')
-.controller('rxPaginateUiCtrl', function ($scope, rxPageTracker) {
+.controller('tableUiPaginationCtr', function ($scope, rxPageTracker) {
```
Comment (CITguy) [inline src/elements/Tables/docs/examples/table.uiPagination.js]: Please correct misspelling (missing ending `l` in `Ctrl`).

File: src/elements/Tables/scripts/rxBatchActions.js [hunk 30]
```diff
@@ -41,7 +41,7 @@ angular.module('encore.ui.elements')
         link: function (scope, element, attrs, controllers) {
 
             var rxBulkSelectCtrl = controllers[0],
-                rxFloatingHeaderCtrl = controllers[1];
+                rxFloatingHeader = controllers[1];
```
Comment (CITguy) [inline src/elements/Tables/scripts/rxBatchActions.js]: Please revert changes to this file. Renames should not take place in `**/scripts/*` directories.

File: src/elements/Tables/scripts/rxBulkSelectMessage.js [hunk 31]
```diff
@@ -36,7 +36,7 @@ angular.module('encore.ui.elements')
                 // Optional controller, so mock it out if it's not present
                 // https://github.com/eslint/eslint/issues/5537
                 // eslint-disable-next-line object-curly-spacing
-                rxFloatingHeaderCtrl = controllers[1] || { update: function () {} };
+                rxFloatingHeader = controllers[1] || { update: function () {} };
```
Comment (CITguy) [inline src/elements/Tables/scripts/rxBulkSelectMessage.js]: Please revert changes to this file. Renames should not take place in `**/scripts/*` directories.

File: src/elements/Forms/docs/rxCharacterCount.js (hunk 22 above)
Comment (cathysiller) [inline src/elements/Forms/docs/rxCharacterCount.js]: updated

File: src/elements/Forms/docs/rxDatePicker.js (hunk 23 above)
Comment (cathysiller) [inline src/elements/Forms/docs/rxDatePicker.js]: updated

File: src/elements/Forms/docs/rxTimePicker.js (hunk 25 above)
Comment (cathysiller) [inline src/elements/Forms/docs/rxTimePicker.js]: updated

File: src/elements/Modals/docs/Modals.docs.html (hunk 26 above)
Comment (cathysiller) [inline src/elements/Modals/docs/Modals.docs.html]: updated

File: src/elements/Modals/docs/Modals.docs.html (hunk 27 above)
Comment (cathysiller) [inline src/elements/Modals/docs/Modals.docs.html]: updated

File: src/elements/Tables/docs/examples/table.uiPagination.html (hunk 28 above)
Comment (cathysiller) [inline src/elements/Tables/docs/examples/table.uiPagination.html]: updated

File: src/elements/Tables/docs/examples/table.uiPagination.js (hunk 29 above)
Comment (cathysiller) [inline src/elements/Tables/docs/examples/table.uiPagination.js]: updated

File: src/elements/Tables/scripts/rxBatchActions.js (hunk 30 above)
Comment (cathysiller) [inline src/elements/Tables/scripts/rxBatchActions.js]: updated

File: src/elements/Tables/scripts/rxBulkSelectMessage.js (hunk 31 above)
Comment (cathysiller) [inline src/elements/Tables/scripts/rxBulkSelectMessage.js]: reverted

File: src/elements/Tables/docs/examples/table.floatingHeader.js [hunk 32]
```diff
@@ -1,5 +1,6 @@
 angular.module('demoApp')
-.controller('rxFloatingHeaderCtrl', function ($scope) {
+.controller('tableFloatingHeaderCtrl', function ($scope) {
+    $scope.searchText = '';
```
Comment (CITguy) [inline src/elements/Tables/docs/examples/table.floatingHeader.js]: This line is unnecessary. Probably overlooked in the rebase.

Comment (cathysiller) [inline src/elements/Tables/docs/examples/table.floatingHeader.js]: removed

**Comment by parlarjb** [comment]
Vidyo LGTM
